app/controllers/pages_controller.rb

```diff
@@ -27,6 +27,13 @@ def robots
     render "pages/robots", layout: false, content_type: "text/plain"
   end
 
+  StaticPage.all.each do |page|
+    define_method page.slug.underscore do
+      @body = page.body
+      render "pages/static"
+    end
+  end
+
```

Comment:
Il faudrait idéalement éviter la méta-programmation et utiliser une route dynamique avec un paramètre, surtout que cette méta-programmation n'est pas répliquée dans `config/routes`

```suggestion
  def show
    page = StaticPage.where(slug: params[:slug]).first
    @body = page.body
    render "pages/static"
  end

```

Reply:
Très bon point @EmCousin 
Je propose qu'on ship en l'état et qu'on fasse ces améliorations dans une autre PR